src/main/java/frc/trigon/robot/subsystems/swerve/Swerve.java

```diff
@@ -0,0 +1,77 @@
+package frc.trigon.robot.subsystems.swerve;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.math.kinematics.ChassisSpeeds;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class Swerve extends SubsystemBase {
+    private final static Swerve INSTANCE = new Swerve();
+
+    public Swerve() {
+        SwerveModule[] swerveModules = SwerveConstants.SWERVE_MODULES;
+        zeroHeading();
+    }
+
+    public static Swerve getInstance() {
+        return INSTANCE;
+    }
+
+    /****
+     * @param translation the x and y
+     * @param rotation the rotation of the robot
+     * robot drives in first person
+     * ****/
```

Comment:
Explain what the function does and fix the formatting

Reply:
idk a better way to expalin it